deployments/k8s/deployment.yaml

```diff
@@ -157,13 +157,22 @@ metadata:
   namespace: accuknox-agents
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: discovery-engine-role
+rules:
```

Comment:
As per my understanding, this changes involve removing DE to run in admin mode/role and make it possible to run in non-admin role with identified permissions. 

Kindly handle this in a separate PR as this will be easy to handle.

Reply:
Would agree :+1: